pkg/amphoracontrollers/flavors.go

```diff
@@ -246,11 +260,14 @@ func ensureFlavors(osclient *openstack.OpenStack, log *logr.Logger, instance *oc
 			log.Info(fmt.Sprintf("Creating Octavia flavor \"%s\"", flavorCreateOpts.Name))
 			_, err := flavors.Create(lbClient, flavorCreateOpts).Extract()
 			if err != nil {
-				return "", err
+				return "", fmt.Errorf("error creating flavor \"%s\": %w", flavorCreateOpts.Name, err)
 			}
 		}
+		flavorSuccess = true
+	}
+	if !flavorSuccess {
+		return "", fmt.Errorf("none of the Octavia flavors could be configured because of errors. last error: %w", err)
```

Comment:
it doesn't seem to work as expected:

`error while creating flavors: none of the Octavia flavors could be configured because of errors. last error: %!w(<nil>)`

Reply:
The issue with this PR is that it floods the output with errors and backtraces until an image is uploaded.
A solution would be to create the amphora flavor profile without amp_image_tag, it would pass and use the default tag from octavia.conf

Reply:
I'll try that. Also I set `ctrl.Result{RequeueAfter: time.Duration(180) * time.Second}, nil` to avoid flooding the output with errors should they occur.